spring-web/src/main/java/org/springframework/http/converter/ResourceRegionHttpMessageConverter.java

```diff
@@ -227,14 +219,14 @@ private void writeResourceRegionCollection(Collection<ResourceRegion> resourceRe
 					println(out);
 				}
 				long resourceLength = region.getResource().contentLength();
-				end = Math.min(end, resourceLength - inputStreamPosition - 1);
```

Comment:
`min,max,modulo` its all used very often from `Math` of course, as this is the happy path all around.